src/app/modules/main/wallet_section/send/controller.nim

```diff
@@ -81,10 +80,10 @@ proc getWalletAccounts*(self: Controller): seq[wallet_account_service.WalletAcco
   return self.walletAccountService.getWalletAccounts()
 
 proc getChainIds*(self: Controller): seq[int] =
-  return self.networkService.getCurrentNetworks().map(n => n.chainId)
+  return self.networkService.getCurrentNetworksChainIds()
 
 proc getEnabledChainIds*(self: Controller): seq[int] =
-  return self.networkService.getCurrentNetworks().filter(n => n.isEnabled).map(n => n.chainId)
+  return self.networkService.getEnabledChainIds()
```

Comment:
If I understand correctly, method like this `getEnabledChainIds` is exposed to the ui, but not used there. Maybe we could remove it completely here?